### PR TITLE
[ptbr] Translate i18n strings used in tags

### PR DESCRIPTION
**Pull request by anacunha**
Signed-off-by: Ana Cunha <1771610+anacunha@users.noreply.github.com>

[Portuguese] Translate i18n strings to use tags #1048 

Changed translation from `"todos"` to `"todas"` as it is meant to be used in sentences such as `"All Tags"`, which translates better to `"Todas Tags"`

### CODE REVIEW

**Review by edsoncelio — APPROVED** [review]
LGTM

**Review by brunoguidone — APPROVED** [review]
LGTM

**Review by jessicalins — APPROVED** [review]
looks good!

**Comment by edsoncelio** [comment]
@jihoon-seo @seokho-son could you take a look on it? It's just to update some tags to the ptbr content, thanks!

**Review by jihoon-seo — APPROVED** [review]
This PR has gained multiple approvals from `pt-br` approvers.

**Pull request merged**
